addons/explosives/CfgAmmo.hpp

```diff
@@ -29,12 +29,15 @@ class CfgAmmo {
     */
     class DirectionalBombBase;
     class ClaymoreDirectionalMine_Remote_Ammo: DirectionalBombBase {
-        ACE_Explosive = "ClaymoreDirectionalMine_Remote_Ammo_Scripted";
+        GVAR(magazine) = "ClaymoreDirectionalMine_Remote_Mag";
+        GVAR(Explosive) = "ClaymoreDirectionalMine_Remote_Ammo_Scripted";
         GVAR(defuseObjectPosition[]) = {0, 0, 0.038};
         soundActivation[] = {"", 0, 0, 0};
         soundDeactivation[] = {"", 0, 0, 0};
     };
-    //class ClaymoreDirectionalMine_Remote_Ammo_Scripted: ClaymoreDirectionalMine_Remote_Ammo;
+    class ClaymoreDirectionalMine_Remote_Ammo_Scripted: ClaymoreDirectionalMine_Remote_Ammo {
+        GVAR(magazine) = "ClaymoreDirectionalMine_Remote_Mag";
```

Comment:
Ah, yes, you're right. I added that because I thought there was something that didn't work properly. Turns out it was the wrong config in getConfig ... -.-